.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: '[FEATURE]'
+labels: enhancement
+assignees: ''
+
+---
+
+**Please note by far the quickest way to get a new feature is to file a Pull Request.**
```

Comment:
We could comment this out right?

Reply:
good idea, not sure if it will show up in the template but we can try. I will update and merge and see what happens.